src/FormsSubmit.vue

```diff
@@ -21,7 +21,7 @@
   -->
 
 <template>
-	<NcContent app-name="forms">
+	<NcContent app-name="forms" :class="{'app-forms-embedded': isEmbedded}">
 		<Submit :form="form"
 			:public-view="true"
 			:share-hash="shareHash"
```

Comment:
```suggestion
			:share-hash="shareHash"
			:sidebar-opened="!isEmbedded"
```
To prevent the follow error:

![Screenshot_20230728_140444](https://github.com/nextcloud/forms/assets/1079143/8ada673d-0c10-4014-a4dd-6d748954f05a)

Reply:
Nice catch!